package.json

```diff
@@ -31,7 +31,7 @@
     "setup:git-hooks": "./scripts/pre-install",
     "start": "NODE_ENV=development npm run storybook --root ./ ",
     "storybook": "start-storybook -p 6006",
-    "test": "jest",
+    "test": "npx jest",
```

Comment:
needed to change this as else my pre-commit hook said 

```
sh: jest: command not found
```

maybe something with newer npm-versions? 🤷